client/verta/verta/deployment/_deployedmodel.py

```diff
@@ -255,6 +255,14 @@ def predict(self, x, compress=False, max_retries=5, always_retry_404=True, alway
         _utils.raise_for_http_error(response)
 
 
+def prediction_input_df(func):
+    import pandas as pd
+
+    def prediction(self, records):
+        return func(self, pd.DataFrame.from_records(records))
```

Comment:
I _think_ the import would be better in the inner function. Not sure though

Reply:
Pretty sure you're right. Fixed.